ethereum_bridge/src/storage/eth_bridge_queries.rs

```diff
@@ -396,6 +400,32 @@ where
         )
     }
 
+    /// Query the Bridge [`ValidatorSetArgs`] at the given [`Epoch`].
+    /// Also returns a map of each validator's voting power.
+    #[inline]
+    pub fn get_bridge_validator_set(
+        self,
+        epoch: Option<Epoch>,
+    ) -> (ValidatorSetArgs, VotingPowersMap) {
+        self.get_validator_set_args(
+            epoch,
+            |&EthAddrBook { hot_key_addr, .. }| hot_key_addr,
+        )
+    }
+
+    /// Query the Governance [`ValidatorSetArgs`] at the given [`Epoch`].
+    /// Also returns a map of each validator's voting power.
+    #[inline]
+    pub fn get_governance_validator_set(
```

Comment:
These two should always return the same thing, no?

Reply:
no, one returns hot key addrs (bridge), the other returns cold key addrs (governance)